packages/backend/src/services/analysis.ts

```diff
@@ -89,12 +89,7 @@ export const runAnalysis = async (sdk: SDK<never, BackendEvents>) => {
           hasAccess: false,
           status: "Untested",
         };
-  
-  
-        if (currentRule.status !== "Untested") {
-          return currentRule;
-        }
-  
```

Comment:
Wonder why this was here. Seems to be related to performance optimization. We're sure we want to remove this?

https://github.com/caido-community/authmatrix/commit/b2645a2801f64b9c4ff4338bea5b44952940dec4